gapic/src/main/com/google/gapid/models/Settings.java

```diff
@@ -264,8 +262,8 @@ private void updateTo(Properties properties) {
     properties.setProperty("updates.lastCheck", Long.toString(lastCheckForUpdates));
     properties.setProperty("updates.available", Boolean.toString(updateAvailable));
     properties.setProperty("analytics.clientId", analyticsClientId);
-    properties.setProperty(
-        "replay.disableOptimization",  Boolean.toString(disableReplayOptimization));
+    properties.setProperty("replay.disableOptimization",
```

Comment:
If possible, I like to avoid line breaks from breaking apart argument lists. The style guide says to keep related things together.